### PR TITLE
Contributors pivot

### PR DESCRIPTION
**Pull request by aalok05**
Added a pivot which will have a list of all contributors to Podcasts repo. Also added my name as the first contributor :)

### CODE REVIEW

File: Podcasts/Pages/AboutPage.xaml [hunk 1]
```diff
@@ -75,58 +75,83 @@
...
-                                 HorizontalAlignment="Center"
-                                 NavigateUri="https://github.com/deltakosh/Podcasts/issues" />
-                <HyperlinkButton x:Uid="PrivacyPolicy" NavigateUri="http://www.urzagatherer.com/Privacy Policy.Podcasts.htm" HorizontalAlignment="Center" Margin="0,-10,0,20"/>
-            </StackPanel>
-        </ScrollViewer>
+                </ScrollViewer>
+            </PivotItem>
+            <PivotItem x:Uid="ContributorsHeader" Margin="0">
+                <ScrollViewer x:Name="ContributorsScrollViewer" VerticalScrollMode="Auto" VerticalScrollBarVisibility="Auto" Padding="15,20,20,0" Grid.Row="1">
+                    <StackPanel MinWidth="500" MaxWidth="500" HorizontalAlignment="Left" x:Name="ContributorsPanel">
```
Comment (deltakosh) [inline Podcasts/Pages/AboutPage.xaml:141]: you need to check the visual states as well to change alignment and fonts

**Review by deltakosh — CHANGES_REQUESTED** [review]

File: Podcasts/Pages/AboutPage.xaml (hunk 1 above)
Comment (aalok05) [reply Podcasts/Pages/AboutPage.xaml:141]: I'm on it.

Comment (aalok05) [reply Podcasts/Pages/AboutPage.xaml:141]: Done :thumbsup:

**Comment by deltakosh** [comment]
Rocks! thank you

**Pull request merged**